src/repository/storage/postgres/migrations/migrate.ts

```diff
@@ -0,0 +1,35 @@
+import pg, { type ClientConfig } from 'pg';
+import { migrate } from 'postgres-migrations';
+import config from './../../../../infrastructure/config/index.js';
```

Comment:
Let's use alias here

Reply:
aliases are not working there since it is a separate process working through `ts-node`. I have not managed to make them work, we can setup them up later